lib/money/money.rb

```diff
@@ -524,12 +525,15 @@ def to_money(given_currency = nil)
   #   Money.new(2000, "USD").exchange_to("EUR")
   #   Money.new(2000, "USD").exchange_to("EUR") {|x| x.round}
   #   Money.new(2000, "USD").exchange_to(Currency.new("EUR"))
-  def exchange_to(other_currency, &rounding_method)
+  def exchange_to(other_currency, rate: nil, &rounding_method)
```

Comment:
what do you think, should we do this, or have an additional method as we have for the bank?

Reply:
why would we want a separate method.

Reply:
I was just thinking it might provide a more explicit interface than adding another option. @antstorm any thoughts?